### PR TITLE
Invariants, error code, use constant for maxAmt

### PR DESCRIPTION
**Pull request by shawnxie999**
### Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Refactor (non-breaking change that only restructures code)
- [ ] Tests (you added tests for code that already exists, or your new feature included in this PR)
- [ ] Documentation update
- [ ] Chore (no impact to binary, e.g. `.gitignore`, formatting, dropping support for older tooling)
- [ ] Release

### API Impact

- [ ] Public API: New feature (new methods and/or new fields)
- [ ] Public API: Breaking change (in general, breaking changes should only impact the next api_version)
- [ ] `libxrpl` change (any change that may affect `libxrpl` or dependents of `libxrpl`)
- [ ] Peer protocol change (must be backward compatible or bump the peer protocol version)

### CODE REVIEW

File: src/ripple/ledger/impl/View.cpp [hunk 1]
```diff
@@ -1738,7 +1738,7 @@ rippleMPTCredit(
 
             // TODO: Replace maxAmt const with a variable
```
Comment (gregtatcam) [inline src/ripple/ledger/impl/View.cpp]: Is this comment needed?

File: src/ripple/app/tx/impl/InvariantCheck.cpp [hunk 2]
```diff
@@ -815,6 +815,10 @@ ValidMPTIssuance::visitEntry(
             mptIssuancesDeleted_++;
         else if (!before)
             mptIssuancesCreated_++;
+
+        if ((*after)[sfOutstandingAmount] >
```
Comment (gregtatcam) [inline src/ripple/app/tx/impl/InvariantCheck.cpp]: This is checked in `rippleMPTCredit`. Why is it needed as invariant?

File: src/ripple/ledger/impl/View.cpp (hunk 1 above)
Comment (shawnxie999) [reply src/ripple/ledger/impl/View.cpp]: Will remove

File: src/ripple/app/tx/impl/InvariantCheck.cpp (hunk 2 above)
Comment (shawnxie999) [reply src/ripple/app/tx/impl/InvariantCheck.cpp]: I think it's cheap and wouldn't hurt to check

Comment (gregtatcam) [reply src/ripple/app/tx/impl/InvariantCheck.cpp]: Well, but it's completely redundant, right? This is why we have the transactor's checks. If "cheap and wouldn't hurt" argument is followed then the invariant might have so many checks that it's no longer cheap. This is just my opinion.

Comment (shawnxie999) [reply src/ripple/app/tx/impl/InvariantCheck.cpp]: fair enough, i dont have a strong opinion on this. Im happy with removing it

Comment (shawnxie999) [reply src/ripple/app/tx/impl/InvariantCheck.cpp]: just pushed a commit to remove it

**Pull request merged**